packages/channel-client/src/types.ts

```diff
@@ -43,6 +53,7 @@ export interface ChannelResult {
   status: ChannelStatus;
   // funding: Funds[]; // do we even need this?
```

Comment:
The api specifies that we return the amount funded to the channel which I think we're not doing right now. I think it's debatable if we need that even.

Reply:
@tomclose what do you think?

Reply:
I think it might be useful to have this in the future, but I'm not averse to removing it until we get to that point.